### PR TITLE
[slave.mk]: remove updategraph.service generation in slave.mk

### PR DESCRIPTION
**Pull request by theasianpianist**
Signed-off-by: Lawrence Lee <t-lale@microsoft.com>

**- What I did**
Currently, updategraph.service is created from the template in files/build_templates at build time and placed in the main directory, but this file is unused and not cleaned up after the build finishes. This is redundant since sonic_debian_extension.sh generates and places a copy inside the built image.

**- Description for the changelog**

* Stop updategraph.service from being redundantly generated inside slave.mk

**- A picture of a cute animal (not mandatory but encouraged)**

### CODE REVIEW

**Comment by jleveque** [comment]
@theasianpianist:Please add more to the description to explain why this change is necessary. Thanks!

File: slave.mk [hunk 1]
```diff
@@ -668,6 +668,7 @@ $(addprefix $(TARGET_PATH)/, $(SONIC_INSTALLERS)) : $(TARGET_PATH)/% : \
 		rm -f $($(docker:-dbg.gz=.gz)_CONTAINER_NAME).sh
 		rm -f $($(docker:-dbg.gz=.gz)_CONTAINER_NAME).service
 	)
+    rm -f updategraph.service
```
Comment (pavel-shirshov) [inline slave.mk]: Please align spaces here

**Comment by jleveque** [comment]
Do we even need to generate updategraph.service here? Is the resulting file used anywhere? It is also generated in https://github.com/Azure/sonic-buildimage/blob/master/files/build_templates/sonic_debian_extension.j2#L198.

File: slave.mk (hunk 1 above)
Comment (theasianpianist) [reply slave.mk]: fixed

**Comment by theasianpianist** [comment]
> Do we even need to generate updategraph.service here? Is the resulting file used anywhere? It is also generated in https://github.com/Azure/sonic-buildimage/blob/master/files/build_templates/sonic_debian_extension.j2#L198.

Actually now that you mention it I'm not sure how it got generated in the main directory in the first place, there's no rule for it or anything. I'm not sure how it got put there on my system, I'll have to take a further look

**Comment by jleveque** [comment]
It's generated on line 647 above (https://github.com/Azure/sonic-buildimage/pull/3153/files/79befc306b83f8465f60f1187dc11d81e4f302da#diff-54252fe595f09e268f275810298a4beeL647). Not sure if this is necessary...

**Comment by theasianpianist** [comment]
You're right, I don't think it is. Looks like sonic_debian_extension.sh places the generate file directly into the image.

**Comment by theasianpianist** [comment]
@jleveque should I just remove it from the Makefile altogether?

**Comment by jleveque** [comment]
If you can prove that it is superfluous to generate it in slave.mk (it may be legacy code that didn't get cleaned up), then yes, you can remove it altogether. Just need to make sure that it gets generated, copied and installed properly in sonic_debian_extension.j2.

**Comment by theasianpianist** [comment]
Confirmed that the updategraph.service generated in slave.mk is unused and can be removed

**Comment by jleveque** [comment]
@theasianpianist: Great! Can you also please update the PR title to match the new nature of the change in this PR?

**Comment by jleveque** [comment]
Retest mellanox, vs please

**Comment by jleveque** [comment]
Retest mellanox please

**Comment by jleveque** [comment]
Retest vs please

**Comment by theasianpianist** [comment]
retest vs please

**Comment by theasianpianist** [comment]
retest vs please

**Comment by theasianpianist** [comment]
retest vs please

**Pull request merged**